src/ui/public/agg_types/__tests__/param_types/_calculate_interval.js

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 import expect from 'expect.js';
 import ngMock from 'ng_mock';
-import VisProvider from 'ui/vis';
+import { VisProvider } from 'ui/vis';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
 import AggTypesParamTypesCalculateIntervalProvider from 'ui/agg_types/param_types/_calculate_interval';
```

Comment:
Hah, interesting. Looks like this test is never being run because it's not mentioned in the index.js file.  I'll remove the file in a separate PR, so the vis team can take a more focused look and in case we ever need to bring it back, it's decoupled from this monster PR. :)